setup.py

```diff
@@ -0,0 +1,58 @@
+from setuptools import Command, setup, find_packages
+import subprocess
+
+# -----------------------------------------------------------------------------
+
+
+def system(command):
+    class SystemCommand(Command):
+        user_options = []
+
+        def initialize_options(self):
+            pass
+
+        def finalize_options(self):
+            pass
+
+        def run(self):
+            subprocess.check_call(command, shell=True)
+
+    return SystemCommand
+
+
+# -----------------------------------------------------------------------------
+
+setup(
+    name='flask-resty-tenants',
+    version='0.1.0',
+    description='Flask Resty Authorization module for multitenancy',
+    url='https://github.com/4Catalyzer/flask-resty-tenants',
+    author='Giacomo Tagliabue',
+    author_email='giacomo.tag@gmail.com',
+    license='MIT',
+    classifiers=(
+        'Development Status :: 3 - Alpha',
+        'Framework :: Flask',
+        'Framework :: Flask-RESTy',
```

Comment:
I don't think this works. I think these have to be in https://pypi.python.org/pypi?%3Aaction=list_classifiers.